Channel.getOrCreate() fails when member has unknown language (lt)

Some of our users have the language `lt` set from the front-end, and we noticed that `Channel.getOrCreate()` call in the back-end fails namely because it's missing from the Language model.

Errors seen:
`Caused by: com.fasterxml.jackson.databind.JsonMappingException: Cannot invoke "com.fasterxml.jackson.annotation.JsonProperty.value()" because the return value of "java.lang.reflect.Field.getAnnotation(java.lang.Class)" is null (through reference chain: io.getstream.chat.java.models.Channel$ChannelGetResponse["members"]->java.util.ArrayList[10]->io.getstream.chat.java.models.Channel$ChannelMember["user"]->io.getstream.chat.java.models.User["language"])`

`Caused by: java.lang.NullPointerException: Cannot invoke "com.fasterxml.jackson.annotation.JsonProperty.value()" because the return value of "java.lang.reflect.Field.getAnnotation(java.lang.Class)" is null`

--- a/src/main/java/io/getstream/chat/java/models/Language.java
+++ b/src/main/java/io/getstream/chat/java/models/Language.java
@@ -64,6 +64,8 @@ public enum Language {
   KA,
   @JsonProperty("ko")
   KO,
+  @JsonProperty("lt")
+  LT,
   @JsonProperty("lv")
   LV,
   @JsonProperty("ms")

